heapster-saw/src/Verifier/SAW/Heapster/RustTypes.hs

```diff
@@ -1573,6 +1573,34 @@ parseFunPermFromRust env w args ret str
 parseFunPermFromRust _ _ _ _ str =
     fail ("Malformed Rust type: " ++ str)
 
+
+-- | Just like `parseFunPermFromRust`, but doesn't take `args` and `ret`.
+-- usefull to translate rust when there is no LLVM module (e.g. from command line)
```

Comment:
Oh, right, I didn't notice that `un3SomeFunPerm` was inside `RustConvM`. It looks like that function could instead be changed to return an `m (SomeFunPerm args ret)` in any monad with a `MonadFail` instance, by using `emptyPPInfo` in place of `rsPPInfo`, since `un3SomeFunPerm` is only ever run at the top level, where the context is empty.